examples/callbacks/src/lib.rs

```diff
@@ -84,4 +84,7 @@ impl RustStringifier {
     }
 }
 
+unsafe impl Send for RustStringifier {}
+unsafe impl Sync for RustStringifier {}
```

Comment:
I'm lightly concerned about this being in an example, where new users might see it and copy-paste it without really understanding the implications. Could you please add a comment about why we need to declare this explicitly rather than Rust figuring it out for us, and why it's safe for us to do so?

Reply:
err - somewhat embarrassingly, I failed to add an `XXX` comment for myself to remember to come back to this, then a month passed and 🤷 . Rust *can* figure it out - I just needed to add a `Sync` bound to the `StoredForeignStringifier` trait - much cleaner. I updated the comment accordingly, but there's less to hand-wave now!